WordPress/Classes/ViewRelated/NUX/LoginViewModel.m

```diff
@@ -361,6 +367,16 @@ - (void)setupObserverationForTheToggleSignInButtonsVisibility
     }];
 }
 
+- (void)setupObservationForApplicationDidEnterBackground
+{
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(didEnterBackground:) name:UIApplicationDidEnterBackgroundNotification object:nil];
```

Comment:
Not a real issue but breaking this up to fit 80 chars would make it easier to read.